functions/pct.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FUNCTIONS_PCT_H
+#define FUNCTIONS_PCT_H
+
+// Usage: Pct<F, V>
+// Gets Percentage V of value F, 
+// Percentages over 100% are allowed and will effectively be a multiplier. 
+// Can be used to multiply or devide any number  
+// F, V: INTEGER
+// return value: INTEGER
+
+class BladeBase;
+
+template<class F, class V>
+class Pct {
```

Comment:
Pct isn't really needed, however, we could easily make an alias for it that just uses Mult<>

Reply:
Indeed, I have made the necessary adjustments